From c539763e9462bad7e4c99cf6331169fb3754e18d Mon Sep 17 00:00:00 2001
From: Rob Browning <rlb@defaultvalue.org>
Date: Mon, 4 Apr 2011 22:46:25 -0500
Subject: Adjust documentation references for Debian.

Various documentation references have been adjusted for Debian.

References to /usr/local/... have been changed to /usr/... as
appropriate, etc.
---
 etc/NEWS |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/etc/NEWS b/etc/NEWS
index 6c0014d..09616a2 100644
--- a/etc/NEWS
+++ b/etc/NEWS
@@ -14,6 +14,11 @@ and NEWS.1-17 for changes in older Emacs versions.
 You can narrow news to a specific version by calling `view-emacs-news'
 with a prefix argument or by typing C-u C-h C-n.
 
+ 
+* Debian specific changes to Emacs
+
+Please see /usr/share/doc/emacs24-common/README.Debian.gz.
+
 
 * Changes in Emacs 24.2
 
